<dec f='linux-5.3.1/include/linux/pci.h' l='427' type='atomic_t'/>
<use f='linux-5.3.1/include/linux/pci.h' l='1151' u='a' c='pci_is_enabled'/>
<offset>12864</offset>
<doc f='linux-5.3.1/include/linux/pci.h' l='427'>/* pci_enable_device has been called */</doc>
<use f='linux-5.3.1/arch/x86/pci/common.c' l='726' u='a' c='pcibios_release_device'/>
<use f='linux-5.3.1/drivers/pci/pci-sysfs.c' l='313' u='a' c='enable_show'/>
<use f='linux-5.3.1/drivers/pci/pci.c' l='1683' u='a' c='pci_enable_device_flags'/>
<use f='linux-5.3.1/drivers/pci/pci.c' l='1700' u='a' c='pci_enable_device_flags'/>
<use f='linux-5.3.1/drivers/pci/pci.c' l='1944' u='a' c='pci_disable_device'/>
<use f='linux-5.3.1/drivers/pci/pci.c' l='1947' u='a' c='pci_disable_device'/>
<use f='linux-5.3.1/drivers/pci/pcie/aer.c' l='897' u='a' c='is_error_source'/>
